src/snaskii.c

```diff
@@ -104,6 +104,11 @@ void play_game(scene_t scenes[GAME_SCENES_SIZE], times_t* times)
 		// TODO: Advance game
 		move_snake(&snake);
 
+		if (!snake.alive) {
+            coord_t energy_blocks[ENERGY_BLOCKS_SIZE];
+            init_game(&snake, energy_blocks);
+		}
```

Comment:
if the snake is not alive does the game display some message informing the snake died or does it just restarts itself?

Reply:
Currently it just restarts. I'm working on a death screen for this situation right now

Reply:
Great